tests/test_fio_workload.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import logging
+import copy
+
+from ocs import defaults, exceptions
+from ocs import workload
+from ocsci.testlib import ManageTest
+from utility.utils import TimeoutSampler
+
+
+from tests.fixtures import (
```

Comment:
You should't have to import fixtures if they are imported into ``conftest.py``.

Reply:
@andrewschoen ,

This is not in conftest.py , so I had to import it